lib/utils/certs.go

```diff
@@ -168,6 +168,9 @@ func VerifyCertificateChain(certificateBytes []byte) error {
 
```

Comment:
Can you add the below as the first thing you do in `VerifyCertificateChain`? This way we won't reject certificates that have leading or trailing white space.

```go
// cleanup the cert so it's easier to process
certificateBytes = bytes.TrimSpace(certificateBytes)
```

This will also allow us to close https://github.com/gravitational/teleport/issues/1349